dockerfiles/Dockerfile

```diff
@@ -62,8 +62,12 @@ COPY . .
 # Generate git version information
 RUN /code/scripts/git_version_info.sh
 
-# Clean up anything we don't need anymore
+# Clean up anything we don't need anymore.
+# Some of these can be purged completely, some of these just remove the package
 RUN apt-get purge -y git curl libcurl4 libcurl3-gnutls && \
+    apt-get remove -y linux-libc-dev && \
+# Keep these packages
+    apt-mark manual libmariadb3 mariadb-common && \
```

Comment:
apt autoremove was removing this along with other packages, so I marked these 2 to not be removed. I didn't see any others that seemed to be a problem with removing.